src/App.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  BrowserRouter, Route, Routes, Navigate,
+} from 'react-router-dom';
+import './App.css';
+import Cookies from 'js-cookie';
+import Login from './components/Login';
+import Signup from './components/Signup';
+import Cars from './components/Cars';
+import ShowCar from './components/ShowBike';
+import AddReservation from './components/AddReservation';
+import AddCars from './components/AddCars';
+import Layout from './components/Layout';
+import ReservationList from './components/ReservationList';
+import DeleteCars from './components/DeleteCars';
+
+function AuthenticatedRoute({ element }) {
+  const isCookieSet = Cookies.get('Authorization') !== undefined;
+  return isCookieSet ? element : <Navigate to="/login" />;
+}
+function App() {
```

Comment:
- [ ] When I click on the hamburger menu, in the mobile view, it won't respond or open the menu:

![image](https://github.com/2004-okari/FC_FRONTEND/assets/57421823/6cf5d59a-3df6-4da4-8b2d-0d5a2e00f618)

Kindly, fix this issue so your mobile app is more usable. 👍